CHANGELOG.md

```diff
@@ -56,13 +56,17 @@
 [Will McGinty](https://github.com/wmcginty)
 [#117](https://github.com/BottleRocketStudios/iOS-Hyperspace/pull/117)
 
-* Make `TransportError` inits `public`..
+* Make `TransportError` inits `public`.
   [Earl Gaspard](https://github.com/earlgaspard)
   [#121](https://github.com/BottleRocketStudios/iOS-Hyperspace/pull/121)
   
   * Create a form URL encoded `HTTP.Body` convenience
   [Will McGinty](https://github.com/wmcginty)
   [#125](https://github.com/BottleRocketStudios/iOS-Hyperspace/pull/125)
+  
```

Comment:
Might just be GitHub, but the leading space looks a bit off for both 121 and 125?

Reply:
Fixed